Criterion/Measurement.hs

```diff
@@ -108,6 +108,7 @@ data GCStatistics = GCStatistics
 -- | Try to get GC statistics, bearing in mind that the GHC runtime
 -- will throw an exception if statistics collection was not enabled
 -- using \"@+RTS -T@\".
+-- If you need guaranteed up-to-date stats, call performGC first.
```

Comment:
Put singlequotes around `performGC` (i.e., `'performGC'`) so that Haddock will link it.